modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -93,6 +93,7 @@
 		"Blueshield",
 		"Command Bodyguard",
 		"Executive Protection Agent",
+		"Henchman",
```

Comment:
Put this in modular_zubbers/code/modules/alternative_job_titles/code/alt_job_titles.dm as

```
/datum/job/blueshield/New()
	alt_titles |= list(
		"Henchman",
		)
	. = ..()
```

Reply:
Changes done as you said, baws!